sofana/R3BSofFrsAnalysis.cxx

```diff
@@ -281,7 +281,9 @@ void R3BSofFrsAnalysis::Exec(Option_t* option)
             Tof_wTref_S8_Cave = hitsingletcal->GetRawTofNs_FromS8();
     } // Loop over sci hits
     //
-    Double_t beta[fNbTof + 1] = { NAN };
+    // Double_t beta[fNbTof + 1] = { NAN };
+    std::vector<Double_t> beta(fNbTof + 1);
```

Comment:
I feel defining a fixed length vector would not fit with the following push_back command as it will add another element on top of the length of fNbTof + 1. Can it be just `std::vector<Double_t> beta;` ?